commcare-salesforce-jobs/test_jobs/TEST-Upsert-Person-Case-Update.js

```diff
@@ -141,6 +168,9 @@ upsert(
         field('Telephone__c', dataValue('form.Status.updated_phone_number')),//need to add a case
         field('CommCare_HH_Code__c', dataValue('indices.parent.case_id')),
         field('Client_Status__c', dataValue('properties.Client_Status')),
+        field('Catchment__c', dataValue('catchmentNewId')),
+        field('Area__c', dataValue('areaNewId')),
+        field('Household_Village__c', dataValue('form.village')),
```

Comment:
@ritazagoni I think the this should be updated as follows because it looks like `village` is _not_ available in [this form's message](https://www.openfn.org/projects/miraclefeet/messages/062613a7-7fbb-7110-8ab6-b77087d1006f). 
```
field('Household_Village__c', dataValue('villageNewId')),
```